pkg/secrets/secrets.go

```diff
@@ -187,9 +183,63 @@ func SecretMounts(filePath, mountLabel, containerWorkingDir string) ([]rspec.Mou
 
 		mounts = append(mounts, m)
 	}
+
+	// Add FIPS mode secret
```

Comment:
Instead of calling this from `SecretMounts()` and needing to check to make sure we aren't adding two FIPS secrets, can you make `AddFIPSModeSecret()` public and call it directly from libpod after we get the normal secrets?

Reply:
That is what I was initially doing, but I spoke with @rhatdan and he pointed out that all secrets mounting should happen in one function, so it is just one thing we need to call to get all the secrets into the container. Thus the code is under `SecretMounts`, which makes sense to me.
Actually I can rework the code a bit to clean it up.

Reply:
Reworked it. PTAL